### ISSUE TITLE
Distinguish "security" from "privacy" considerations

### ISSUE THREAD

**Issue by jyasskin** [mikewest/spec-questionnaire#7]
Specs can have a "Security and Privacy Considerations" section or one or both of "Security Considerations" and "Privacy Considerations". It'd be nice if either
1. It were clear that we should just write a "Security and Privacy Considerations" section and ignore any possible distinction between security vs privacy issues, or
2. The individual sections in the questionnaire said whether they addressed security or privacy.

I'd lean toward (1), since we could bikeshed the categorization forever.

**Comment by marcoscaceres** [comment]
I agree about 1. Reminds me of a funny sign at my local gym, which reads "For the **security** of our members, camera phones are not allowed in the change rooms".

**Comment by tantek** [comment]
Agree 1. Many considerations overlap and it's not particularly useful to make a distinction.
